lib/taskana-core/src/main/java/pro/taskana/impl/ObjectReferenceQueryImpl.java

```diff
@@ -88,13 +94,24 @@ public ObjectReferenceQuery value(String... values) {
             RowBounds rowBounds = new RowBounds(offset, limit);
             result = taskanaEngineImpl.getSqlSession().selectList(LINK_TO_MAPPER, this, rowBounds);
             return result;
+        } catch (Exception e) {
+            if (e instanceof PersistenceException) {
+                if (e.getMessage().contains("Invalid operation: result set is closed.")) {
+                    TaskanaRuntimeException ex = new TaskanaRuntimeException(
+                        "The offset beginning was set over the amount of result-rows.", e.getCause());
+                    ex.setStackTrace(e.getStackTrace());
+                    throw ex;
+                }
+            }
+            throw e;
```

Comment:
Sollten wir nicht alle Exceptions in TaskanaExceptions wrappen?

Reply:
Alle anderen Exceptions die Auftreten könnten sind bereits TaskanaExceptions (NotAuhorized/...). Die Unterscheidung dient nur dem genauen Loggen des Fehlers mit einem zu hohen Offset bei DB2. H2 gibt EMPTY zurück.

Reply:
Message identifizierung wurde an Fehlercode ERRORCODE=-4470 angepasst.